source/magpie/helper.d

```diff
@@ -37,6 +37,19 @@ template getArgsList(args...)
         alias getArgsList = AliasSeq!();
 }
 
+/// Template to evaluate RowType when column is dropped
+template dropper(int[] pos, int rem, Types...)
```

Comment:
It seems the user should always call this with `rem == 0`. Try using a nested template helper to remove that need.

Reply:
Addressed in dec86d1